cura/CuraApplication.py

```diff
@@ -1940,6 +1945,13 @@ def checkIsValidProjectFile(self, file_url):
         try:
             result = workspace_reader.preRead(file_path, show_dialog=False)
             return result == WorkspaceReader.PreReadResult.accepted
+        except FileNotFoundError:
+            result_message = Message(text = self._i18n_catalog.i18nc("@info:status Don't translate the XML tag <filename>!",
+                                                                     "Failed to load <filename>{0}</filename>. No such file or directory.",
```

Comment:
Looks like you wrote this kind of automatically, but I don't think project files can ever be a directory. I'll adjust this message to "File is moved or deleted."